pkgs/applications/science/misc/megam/default.nix

```diff
@@ -20,7 +28,7 @@ stdenv.mkDerivation {
 
   makeFlags = [
     "CAML_INCLUDES=${ocaml}/lib/ocaml/caml"
-    "WITHBIGARRAY=bigarray.cma"
+    ("WITHBIGARRAY=" + lib.optionalString (lib.versionOlder ocaml.version "4.08.0") "bigarray.cma")
```

Comment:
between 4.08.0 and <5 a warning was triggered because of https://github.com/ocaml/ocaml/pull/1685 but the file existed so it builded fine (your PR). It was definitely removed in 5.0 https://github.com/ocaml/ocaml/pull/10896 and thus causing error error.

Reply:
The version check is no longer necessary if we stick to OCaml 5.0+.

Reply:
If someone need to override for a reason I think it can stay here